src/TConsole.cpp

```diff
@@ -2681,6 +2682,42 @@ bool TConsole::setBackgroundColor(const QString & name, int r, int g, int b, int
 
 }
 
+bool TConsole::raiseWindow(const QString & name )
+{
+    std::string key = name.toLatin1().data();
```

Comment:
As part of future __I18n__ plans to convert the window/label names to be `QString`s there will be no need for this ASCII only conversion and the following line will convert to the simpler: `if( mSubConsoleMap.contains( name )`